pkgs/games/pokete/default.nix

```diff
@@ -41,7 +42,8 @@ python3.pkgs.buildPythonApplication rec {
   passthru.tests = {
     pokete-version = testers.testVersion {
       package = pokete;
-      command = "pokete --help";
+      command = "${faketty}/bin/faketty pokete --help";
+      version = "v${version}";
```

Comment:
doesn't the default version work?

Reply:
the default version picks up the wrong line

> Usage: /nix/store/f2mlz7c7ls0v11mb5jdg54fwbaahvgk6-pokete-0.9.1/share/pokete/pokete.py (<options>)

with this change it picks up the line with the version output

> Pokete Grey Edition v0.9.1